script/prepare-windows-cert.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+GITHUB_CERT_PASSWORD=$1
+DESKTOP_CERT_TOKEN=$2
+
+curl \
+  -H "Authorization: token $DESKTOP_CERT_TOKEN" \
+  -H "Accept: application/vnd.github.v3.raw" \
+  --output windows-certificate.pfx \
+  https://api.github.com/repos/desktop/desktop-secrets/contents/windows-certificate.pfx
+
+openssl pkcs12 -in windows-certificate.pfx -nocerts -nodes -out private-key.pem  -passin pass:${GITHUB_CERT_PASSWORD} || echo "no bueno 1"
+openssl pkcs12 -in windows-certificate.pfx -nokeys -nodes -out certificate.pem -passin pass:${GITHUB_CERT_PASSWORD} || echo "no bueno 2"
```

Comment:
1. `|| echo "no bueno"` doesn't seem like it adds value, but it will definitely shadow the exit status of the `openssl` invocation. Is that intended?
2. These files have no newlines add the end of the file, which is "standard" for vim and some other editors. Did you intentionally stop including that with new files?

Reply:
stray debugging and not intentional